### DISCUSSION TITLE
I am a student and am looking for the student campus option. But I am unable to show proof as my school ID is a school ID, while the county is the one that has the education campus option.

### DISCUSSION THREAD

**Discussion by ghost** [community/community#111046] [GitHub Education]
### Hi everyone,

I am a student and am looking for the student campus option. But I am unable to show proof as my school ID is a school ID, while the county is the one that has the education campus option. So I don't know what to do and how to get it.

**Comment by SebastienXia** [comment]
### Hi, @FazTDM-YT  and welcome to the Github community ! ❤️

>The school ID card is one of the ways to prove that you are a student, there are others, such as:

A 2023-2024 schooling certificate
...

`[close the discussion if this helped you with your problem]`

**Comment by LaLiiSa-28** [comment]
Hi, it won't let me show it too, it says that my school isn't qualified for it.. what do I have to do ? 

Thanks !
